support-lambdas/it-test-runner/build.sbt

```diff
@@ -3,7 +3,7 @@ import com.gu.riffraff.artifact.RiffRaffArtifact.autoImport.riffRaffManifestProj
 import sbt.Keys.libraryDependencies
 
 version := "0.1-SNAPSHOT"
-scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-target:jvm-1.8", "-Xfatal-warnings")
+scalacOptions ++= Seq("-deprecation", "-feature", "-unchecked", "-release:8", "-Xfatal-warnings")
```

Comment:
Used -release:8 instead of  "-target:jvm-1.8"  code/support-frontend/support-lambdas/it-test-runner/build.sbt
because it showed error target is deprecated: Use -release instead to compile against the correct platform API. during compilation